cpp/ps0003/ps0003_soln.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+using namespace std;
+int main()
+{
+  // prompting user for number 2/2
+  int number;
+  cout << "please enter number \n";
+  cin >> number;
+
+  // checking if the number is odd or even 3/3
+  if (number % 2 == 0)
+    cout << "this number is even \n";
+  else
+    cout << "this number is odd\n";
+  
+  return 0;
+}
+// great work 😎 8/10 ⭐
```

Comment:
- [x] program returns `0` as everything is okay
- [ ] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code

you can use ternary operator
```cpp
std::cout << "Number is " << ((x % 2 == 0) ? "even" : "odd") << std::endl
```